proto/proto-backplane-grpc/build.gradle

```diff
@@ -72,6 +72,29 @@ TaskProvider<Task> generateProtobuf = Docker.registerDockerTask(project, 'genera
   }
 }
 
+TaskProvider<Task> generateProtoDocs = Docker.registerDockerTask(project, 'generateProtoDocs') {
```

Comment:
This task is not needed anymore. Additionally, you will need to setup the configuration and artifacts to copy the data out of the container from `/generated/proto-doc`.